templates/taxi/car_detail.html

```diff
@@ -12,7 +12,22 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>Drivers
+    {% if user in car.drivers.all %}
+      <form method="post" action="{% url 'taxi:driver-car-update' car_pk=car.pk %}">
+        {% csrf_token %}
+        <input type="hidden" name="action" value="remove">
+        <button type="submit" class="btn btn-danger link-to-page">Delete me from this car</button>
+      </form>
+      {% else %}
+        <form method="post" action="{% url 'taxi:driver-car-update' car_pk=car.pk %}">
+          {% csrf_token %}
+          <input type="hidden" name="action" value="add">
+          <button type="submit" class="btn btn-primary link-to-page">Assign me to this car</button>
+        </form>
```

Comment:
You can call {% if %} in form after csrf token to avoid unnecessary code repeating